src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Builders/Services/UnitTestOutputLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Xunit.Abstractions;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Tests.Builders.Services;
+
+public class UnitTestOutputLoggerBuilder<T>
```

Comment:
I think a comment at the class level would be helpful here to explain what this is.